CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CovertInfiltrationInfo.uc

```diff
@@ -104,4 +112,44 @@ protected function InitExistingCampaign()
 	{
 		bCompletedFirstOrdersAssignment = true;
 	}
+
+	CurrentBarracksLimit = default.StartingBarracksLimit;
+}
+
+/////////////////////
+/// Barracks Size ///
+/////////////////////
+
+function IncreaseBarracksSizeI(XComGameState UpdateState)
```

Comment:
Please no helper functions like these in `XCGS_CIInfo`, this is supposed to be a dummy data-tracker-only class.

You can make a helper like these in `X2StrategyElement_InfiltrationFacilityUpgrades` though